>[Integration tests (asan, flaky check)](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=75929&sha=9892424e1632cd079e691e9d89f6daeb028bcad7&name_0=PR&name_1=Integration%20tests%20%28asan%2C%20flaky%20check%29) — Failed tests found: error: 0, passed: 1006, fail: 4, skipped: 15, brok

```
E           helpers.client.QueryRuntimeException: Client failed! Return code: 86, stderr: Received exception from server (version 25.2.1):
E           Code: 598. DB::Exception: Received from 172.16.1.2:9000. DB::Exception: Backup File('/backups/test_backup/') already exists. Stack trace:
E           
```

--- a/tests/integration/test_backup_log/test.py
+++ b/tests/integration/test_backup_log/test.py
@@ -1,4 +1,5 @@
 import pytest
+import uuid
 
 from helpers.cluster import ClickHouseCluster
 from helpers.test_tools import TSV
@@ -64,7 +65,8 @@ def test_backup_log():
     instance.query("SYSTEM FLUSH LOGS")
     instance.query("DROP TABLE IF EXISTS system.backup_log SYNC")
 
-    backup_name = "File('/backups/test_backup/')"
+    backup_id = uuid.uuid4().hex
+    backup_name = f"File('/backups/test_backup_{backup_id}/')"
     assert instance.query("SELECT * FROM system.tables WHERE name = 'backup_log'") == ""
 
     backup_id = backup_table(backup_name)

--- a/tests/integration/test_backup_log/test.py
+++ b/tests/integration/test_backup_log/test.py
@@ -22,8 +22,8 @@ def start_cluster():
 
 
 def backup_table(backup_name):
-    instance.query("CREATE DATABASE test")
-    instance.query("CREATE TABLE test.table(x UInt32) ENGINE=MergeTree ORDER BY x")
+    instance.query("CREATE DATABASE IF NOT EXISTS test")
+    instance.query("CREATE TABLE IF NOT EXISTS test.table(x UInt32) ENGINE=MergeTree ORDER BY x")
     instance.query("INSERT INTO test.table SELECT number FROM numbers(10)")
     return instance.query(f"BACKUP TABLE test.table TO {backup_name}").split("\t")[0]
 
@@ -62,7 +62,7 @@ def test_backup_log():
         )
 
     instance.query("SYSTEM FLUSH LOGS")
-    instance.query("drop table system.backup_log")
+    instance.query("DROP TABLE IF EXISTS system.backup_log SYNC")
 
     backup_name = "File('/backups/test_backup/')"
     assert instance.query("SELECT * FROM system.tables WHERE name = 'backup_log'") == ""
@@ -73,7 +73,7 @@ def test_backup_log():
         [["CREATING_BACKUP", 0, 0, 0, ""], ["BACKUP_CREATED", 0, 0, 0, ""]]
     )
 
-    instance.query("DROP TABLE test.table SYNC")
+    instance.query("DROP TABLE IF EXISTS test.table SYNC")
 
     restore_id = restore_table(backup_name)
     instance.query("SYSTEM FLUSH LOGS")

--- a/tests/integration/test_backup_restore_new/test.py
+++ b/tests/integration/test_backup_restore_new/test.py
@@ -1,3 +1,4 @@
+from dataclasses import dataclass
 import glob
 import os.path
 import random
@@ -118,10 +119,19 @@ def get_events_for_query(query_id: str) -> Dict[str, int]:
     return result
 
 
-BackupInfo = namedtuple(
-    "BackupInfo",
-    "name id status error num_files total_size num_entries uncompressed_size compressed_size files_read bytes_read",
-)
+@dataclass
+class BackupInfo:
+    name: str
+    id: int
+    status: str
+    error: str
+    num_files: int
+    total_size: int
+    num_entries: int
+    uncompressed_size: int
+    compressed_size: int
+    files_read: int
+    bytes_read: int
 
 
 def get_backup_info_from_system_backups(by_id=None, by_name=None):

